sys_conn_helper_linux.go

```diff
@@ -17,3 +22,23 @@ const (
 )
 
 const batchSize = 8 // needs to smaller than MaxUint8 (otherwise the type of oobConn.readPos has to be changed)
+
+func forceSetReceiveBuffer(c interface{}, bytes int) error {
```

Comment:
There's no downside to always calling SO_RCVBUFFORCE, is there?

If that's the case, you could just do it as part of `setReceiveBuffer`, no?

Reply:
A couple a reasons to keep it here:
1. `unix.SO_RCVBUFFORCE` is only defined for GOOS=linux. So this would fail if we put it in `setReceiveBuffer` as is, since that's shared.
2. Having it as a separate function lets us test only this thing. However, I'm not sure this will actually work since CI runner might not have CAP_NET_ADMIN (I don't when running this as a normal user).